jenkins-build-pack-website/Jenkinsfile

```diff
@@ -43,22 +43,25 @@ def Event_Status = [
 @Field def env_prefix = ''
 @Field def region = ''
 @Field def jq = '/usr/local/bin/jq'
+@Field def scm_base = ''
 
 node ()  {
 	region = env.jazz_region
 	g_base_url = "https://${env.API_KEY}.execute-api.${region}.amazonaws.com/dev"
 
 	echo "Build Pack website params: " + params
 
-    def service = params.service_name
-	def branch = params.scm_branch
-	def domain = params.domain
-
-    def bitbucket_name = params.service_name
+    def service 		= params.service_name
```

Comment:
can you check why indentation is messed up here?

Reply:
It was the problem with VSC changed the settings appropriately